apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -984,6 +984,14 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
             };
         }
 
+        // Add `scope validators` only if `scope validators` are visible
```

Comment:
Lets add `.` at the end.

Reply:
Fixed with 8535eecb987fefadc2bc7058e3a9070e260baf10